counsel.el

```diff
@@ -2695,6 +2717,15 @@ This uses `counsel-ag' with `counsel-ack-base-command' replacing
 Note: don't use single quotes for the regex."
   :type 'string)
 
+(defcustom counsel-rg-version
+  (when (string-match "\\(?:\\`\\| \\)\\([^ =]+\\) " counsel-rg-base-command)
+    (let* ((rg (match-string 1 counsel-rg-base-command))
+           (version (shell-command-to-string (concat rg " --version"))))
```

Comment:
Evaluating a `defcustom` should never call the shell (or processes in general). Please do this lazily / on demand and cache the result if necessary.